File: commons/src/main/java/com/eirsteir/coffeewithme/commons/dto/UserDetailsDto.java
package com.eirsteir.coffeewithme.commons.dto;

import lombok.*;
import lombok.experimental.Accessors;

@Data
@Builder
@EqualsAndHashCode
@Accessors(chain = true)
@NoArgsConstructor
@AllArgsConstructor
public class UserDetailsDto {

    private Long id;
    private String email;
    private String nickname;
    private String name;
    private Boolean isFriend;
    private Integer friendsCount;

}

File: commons/src/main/kotlin/com/eirsteir/coffeewithme/commons/dto/UserDetailsDto.kt
package com.eirsteir.coffeewithme.commons.dto

data class UserDetailsDto(
    val id: Long? = null,
    val email: String? = null,
    val nickname: String? = null,
    val name: String? = null,
    val isFriend: Boolean? = null,
    val friendsCount: Int? = null,
)